torch/_inductor/codegen/wrapper.py

```diff
@@ -1729,11 +1732,28 @@ def generate_c_shim_extern_kernel_call(self, kernel, args):
         shim_fn = f"aoti_torch_{kernel_suffix}"
         self.writeline(f"AOTI_TORCH_ERROR_CODE_CHECK({shim_fn}({', '.join(args)}));")
 
-    def generate_c_shim_extern_kernel_alloc_call(self, extern_kernel, args):
+    def generate_c_shim_extern_kernel_alloc(self, extern_kernel, args):
```

Comment:
We could use some refactoring for `ir.ExternKernelAlloc` and `ir.FallbackKernel` at some point.